moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -181,6 +181,17 @@ PlanningSceneMonitor::~PlanningSceneMonitor()
   rm_loader_.reset();
 }
 
+planning_scene::PlanningScenePtr PlanningSceneMonitor::copyPlanningScene(const moveit_msgs::PlanningScene& diff)
+{
+  lockSceneRead();  // We cannot use LockedPlanningSceneRO as this requires a PSMPtr
+  planning_scene::PlanningScenePtr scene = planning_scene::PlanningScene::clone(getPlanningScene());
+  unlockSceneRead();
```

Comment:
```suggestion
  // We cannot use LockedPlanningSceneRO for RAII because it requires a PSMPtr
  // Instead assume clone will not throw
  lockSceneRead();
  planning_scene::PlanningScenePtr scene = planning_scene::PlanningScene::clone(getPlanningScene());
  unlockSceneRead();
```